src/registrar/tests/test_models.py

```diff
@@ -1214,6 +1215,68 @@ def test_has_contact_info(self):
         self.user.phone = None
         self.assertFalse(self.user.has_contact_info())
 
+    def test_has_portfolio_permission(self):
+        """
+        0. Returns False when user does not have a permission
+        1. Returns False when a user does not have a portfolio
+        2. Returns True when user has direct permission
```

Comment:
**suggestion:** These all really sound like different tests. do they all have to be in this one unit test as opposed to breaking them out?

Reply:
THey don't have to be in the same test.  Just did so for simplicity